README.md

```diff
@@ -105,22 +105,24 @@ you want these variants to be:
 #### Performance Build Variants
 For accurate performance measurements, read this section!
 
-If you want to analyze performance during **local development** (note: there is a non-trivial performance impact - see caveats):
-- Recommendation: use a debuggable variant (see "local.properties helpers" below) with local Leanplum, Adjust, & Sentry API tokens: contact the front-end perf group for access to them
-- Rationale: There are numerous performance-impacting differences between debug and release variants so we need a release variant. To profile, we also need debuggable, which is disabled by default for release variants. If API tokens are not provided, the SDKs may change their behavior in non-trivial ways.
-- Caveats:
-  - debuggable has a non-trivial & variable impact on performance but is needed to take profiles.
-  - Random experiment opt-in & feature flags may impact performance (see [perf-frontend-issues#45](https://github.com/mozilla-mobile/perf-frontend-issues/issues/45) for mitigation).
-  - This is slower to build than debug builds because it does additional tasks (e.g. minification) similar to other release builds
+Before heading into performance, read this [quick guide on performance with Fenix](https://wiki.mozilla.org/Performance/How_to_get_started_on_Fenix)
+
+If you want to analyze performance during **local development** :
+- Use a debuggable build (** the only option right now is the debug variant. However, add `debuggable=true`, which will make it debuggable**) if you want to:
+  1) Profile using Android Studio profiler
+  2) See what is going on in different threads (i.e: the I/O thread)
+- Use a production builds if you want:
+  1) A more accurate representation of the application with how it would run normally.
 
 If you want to run **performance tests/benchmarks** in automation or locally:
-- Recommendation: production builds. If debuggable is required, use recommendation above but note the caveat above. If your needs are not met, please contact the front-end perf group to identify a new solution.
-- Rationale: like the rationale above, we need release variants so the choice is based on the debuggable flag.
+- Use production build since it is much closer in behavior compared to what users see in the wild.
 
 For additional context on these recommendations, see [the perf build variant analysis](https://docs.google.com/document/d/1aW-m0HYncTDDiRz_2x6EjcYkjBpL9SHhhYix13Vil30/edit#).
 
 Before you can install any release variants, **you will need to sign them:** see [Automatically signing release builds](#automatically-sign-release-builds) for details.
 
+**known disabled-by-default features**
+In production builds, Fenix will use some features that are automatically turned off in debug builds ( A good example of that is telemetry). This may cause some slight difference between productions and debuggable builds, but for most use cases, this should be negligible.
```

Comment:
I think you should start by telling devs why they should care that things are disabled, e.g. "Some features are disabled by default; this can be problematic when performance testing because you want to see how your code interacts with it. The known features that are disabled-by-default are Sentry, Leanplum, ... For debug builds only, telemetry is disabled by default."